server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/table.ftl

```diff
@@ -44,7 +44,7 @@
                 <th title="Key/value pairs over each instance, table or tablet.">Entries&nbsp;</th>
                 <th title="The number of Key/Value pairs inserted. (Note that deletes are considered inserted)">Ingest&nbsp;</th>
                 <th title="The number of key/value pairs returned to clients. (Not the number of scans)">Query&nbsp;</th>
-                <th title="The amount of time that ingest operations are suspended while waiting for data to be written to disk.">Hold&nbsp;Time&nbsp;</th>
+                <th title="The amount of time live ingest (mutations, batch writer) is suspended while waiting for tserver memory to free up. Writes are attempting to exceed tserver.memory.maps.max">Hold&nbsp;Time&nbsp;</th>
```

Comment:
I like the first sentence, but I suggest: `s/is suspended/has been suspended/`.

I'm not sure I understand what the second sentence is trying to say. It may not be necessary. Or maybe it would be simpler to say something about the tserver awaiting a flush / minc?

Reply:
> I'm not sure I understand what the second sentence is trying to say. It may not be necessary. Or maybe it would be simpler to say something about the tserver awaiting a flush / minc?

I was trying to describe why it has a hold time in as few words possible.  This seems to be the only spot in the code where we decide to hold commits up: https://github.com/apache/accumulo/blob/62110a3c27b9a27d8a7c0bc272aa6778910acad7/server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java#L575

Reply:
And provide a pointer to the configuration property determining when to hold.

Reply:
Reading the docs for that property implies that it's not so simple as just this property. How, and by which properties, memory is managed, is complicated and can change over time, as that's implementation.

It might be better to stick to a high-level description, rather than pointing to specific properties / implementations, and explain that it's simply being held because its buffers are full and it needs to free up memory by flushing / minor compacting.

Also, these are just tool-tips, not supposed to be complete docs. I think "The duration over which commits have been held, waiting for the tserver to clear up memory."

I have reservations on your wording, because it didn't seem clear to me, and might be misleading, but I don't know what's best.

Reply:
I wanted to make more clear to admin what is happening for this number to be high.  To make it clear that live ingest is held up vs bulk imports.  

What about "The duration live ingest ops (mutations, batch writes) have been held while waiting for tserver to free up memory"

Reply:
Or "The amount of time live ingest operations (mutations, batch writes) have been waiting for the tserver to free up memory"

Reply:
Either of those would be fine... I didn't mean to omit the part that emphasized live ingest in my suggestion. I only meant to drop the part implying that a specific single property was the culprit, from the second sentence.

The only part I thought should be changed for the first sentence is to make it "past-tense and ongoing", rather than imply "present tense and instantaneous" ("has/have been suspended" vs. "is suspended"). Both of your last two suggestions encapsulate what I was going for there.